docs/mutations.rst

```diff
@@ -97,11 +100,13 @@ Imagine a mutation for creating ``Discussion`` that takes category, poster, titl
 
     type_def = """
         type Mutation {
-            createDiscussion(category: ID!, title: String!, isAnnouncement: Boolean, isClosed: Boolean) {
-                status: Boolean!
-                error: Error
-                discussion: Discussion
-            }
+            createDiscussion(category: ID!, title: String!, isAnnouncement: Boolean, isClosed: Boolean): DiscussionPayload
+        }
+
+        type DiscussionPayload {
```

Comment:
IMHO, we shouldn't add payload for this (and next)  example - we've already explained payload concept in the docs, and those examples focused on illustrating `input` usage.